casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala

```diff
@@ -73,11 +74,11 @@ class MultiParentCasperAddBlockSpec extends FlatSpec with Matchers with Inspecto
 //  }
 
   it should "accept signed blocks" in effectTest {
-    TestNode.standaloneEff(genesis).use { node =>
+    TestNode.standaloneEff(genesis, shardId = SHARD_ID).use { node =>
```

Comment:
Can TestNode read shardId from supplied `genesis` context (genesisBlock.shardId), instead of adding additional parameter?